var/spack/repos/builtin/packages/sourmash/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Sourmash(PythonPackage):
+    """Sourmash: Quickly search, compare, and analyze genomic and metagenomic data sets with
+    k-mer sketches."""
+
+    homepage = "https://sourmash.bio/"
+    pypi = "sourmash/sourmash-4.8.2.tar.gz"
+
+    version("4.8.2", sha256="e0df78032e53ed88977445933ba3481dd10c7d3bd26d019511a6a4e6d7518475")
+
+    depends_on("python@3.8:")
```

Comment:
```suggestion
    depends_on("python@3.8:", type=("build", "run"))
```
Unless you also need to link to it